pulp_container/app/token_verification.py

```diff
@@ -95,10 +97,10 @@ def _build_authenticate_string(self, source_path):
         Build a formatted authenticate string.
 
         For example, A created string is the following format:
-        realm="https://token",service="docker.io",scope="repository:my-app:push".
+        realm="https://token:123",service="docker.io",scope="repository:my-app:push".
```

Comment:
is 123 a port? port usually comes after the hostname

Reply:
what about instead of https://token:123 you write https://<hostname>:<port>/token,